### ISSUE TITLE
feat(worker): add advanced training settings for quality improvement

### ISSUE THREAD

**Issue by raolivei** [raolivei/visage#43]
## Summary
Allow users to tune training parameters for better quality results.

### Training Settings
- [ ] `training_steps`: 800-3000 (default: 1500)
- [ ] `lora_rank`: 16, 32, 64, 128 (default: 32)
- [ ] `learning_rate`: 1e-5 to 1e-4 (default: auto/1e-4)
- [ ] `scheduler`: constant, cosine, linear (default: constant)

### Generation Settings  
- [ ] `num_inference_steps`: 20-50 (default: 30)
- [ ] `guidance_scale`: 3-15 (default: 7.5)
- [ ] `lora_scale`: 0.5-1.0 (default: 0.8)

### Quality Filtering
- [ ] `face_similarity_threshold`: 0.3-0.8 (default: 0.5)
- [ ] `aesthetic_threshold`: 4-7 (default: 5.0)
- [ ] `min_outputs_per_style`: 1-10 (default: 3)

## Presets
```
Fast (15 min):      800 steps, rank 16, 20 inference steps
Balanced (30 min):  1500 steps, rank 32, 30 inference steps  
Quality (60 min):   2500 steps, rank 64, 40 inference steps
Maximum (90 min):   3000 steps, rank 128, 50 inference steps
```

### API Schema
```python
class TrainingSettings(BaseModel):
    steps: int = Field(1500, ge=500, le=5000)
    lora_rank: Literal[16, 32, 64, 128] = 32
    learning_rate: float = Field(1e-4, ge=1e-6, le=1e-3)
    
class GenerationSettings(BaseModel):
    inference_steps: int = Field(30, ge=10, le=100)
    guidance_scale: float = Field(7.5, ge=1.0, le=20.0)
    lora_scale: float = Field(0.8, ge=0.1, le=1.5)
```

### UI
- Training presets dropdown (Fast/Balanced/Quality/Maximum)
- 'Advanced' toggle for manual parameter control
- Estimated time based on settings

## Related
- Part of commercial quality initiative
